src/assignments/orbit-report.rst

```diff
@@ -423,8 +503,26 @@ Copy the URL for your Github repository and paste it into the box, then click
 
 .. _orbit-report-demo:
 
-Application Demo
-----------------
+Orbit Report Demo
+------------------
+
+Once you complete all of the tasks outlined above, your project should behave
+something like this:
+
+Sorting the Table
+^^^^^^^^^^^^^^^^^^
+
+.. figure:: figures/orbit-report-sort.gif
+   :alt: Sorting the table gif.
+
+Searching the Table
```

Comment:
minor point but the orbit count shows up in this gif

Reply:
Yes, I had the component active while I was making the GIF, and it jumped into the view when the table length decreased.